.env.tpl

```
@@ -0,0 +1,16 @@
+# set these to your upload API service URL and the DID your service is using as its service DID
+NEXT_PUBLIC_W3UP_SERVICE_URL=https://staging.up.web3.storage
+NEXT_PUBLIC_W3UP_RECEIPTS_URL=https://staging.up.web3.storage/receipt/
+NEXT_PUBLIC_W3UP_SERVICE_DID=did:web:staging.web3.storage
+NEXT_PUBLIC_W3UP_PROVIDER=did:web:staging.web3.storage
+NEXT_PUBLIC_IPFS_GATEWAY_URL=https://%ROOT_CID%.ipfs-staging.w3s.link
```

I think environment variable set is different from workflow, should we change to `STORACHA`?
Workflow: `NEXT_PUBLIC_STORACHA_XXXX`

```suggestion
NEXT_PUBLIC_STORACHA_SERVICE_URL=https://staging.up.web3.storage
NEXT_PUBLIC_STORACHA_RECEIPTS_URL=https://staging.up.web3.storage/receipt/
NEXT_PUBLIC_STORACHA_SERVICE_DID=did:web:staging.web3.storage
NEXT_PUBLIC_STORACHA_PROVIDER=did:web:staging.web3.storage
```